tools/image_processing/yolo/yolo_training.xml

```diff
@@ -0,0 +1,154 @@
+	    <param name="ds_name" value="tail" />
+	    <section name="training_params">
+                <param name="num_train" value="70" />
+	        <param name="num_class" value="1" />
+	        <param name="model_name" value="yolo11n-seg" />
+                <param name="epochs" value="10" />
+                <param name="image_size" value="500" />
+                <param name="scale" value="0.8" />
+                <param name="degree" value="10.0" />
+                <param name="hsv_v" value="0.5"/>
+                <param name="init_lr" value="0.02" />
+                <param name="weight_decay" value="0.001" />
+            </section>
+	    <output_collection name="output_files" type="list" count="4">
+		<element name="results_metrics">
```

Comment:
how many outputs are in here? Should we separate the text files from the plots? Maybe 4 different outputs, than everything in one collection?

Reply:
yes 4 outputs for now.

I also think 4 different outputs may be better.

there are also other intermediate images as result of training, a lot, but for us they are not needed. I'm just thinking if in future other users want to tweak the output to include the intermediate images, a collection maybe easier.